src/draw/lv_draw.c

```diff
@@ -3,6 +3,10 @@
  *
  */
 
+/**
+ * Modified by NXP on 2024
```

Comment:
```suggestion
 * Modified by NXP in 2024
```

(Same for the others)